OracleWebLogic/samples/wls-k8s-domain/README.md

```diff
@@ -0,0 +1,137 @@
+NAME      CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS      CLAIM                    STORAGECLASS   REASON    AGE
+pv1       10Gi       RWX           Recycle         Available                            manual                   17m
+pv2       10Gi       RWX           Recycle         Bound       default/wlserver-pvc-1   manual                   17m
+pv3       10Gi       RWX           Recycle         Bound       default/wlserver-pvc-2   manual                   17m
+
+$ kubectl get pvc
+NAME             STATUS    VOLUME    CAPACITY   ACCESSMODES   STORAGECLASS   AGE
+wlserver-pvc-1   Bound     pv2       10Gi       RWX           manual         18m
+wlserver-pvc-2   Bound     pv3       10Gi       RWX           manual         18m
+```
+We have three pv defined and two pvc defined. One pv is reserved for later use.
+
+#### 4.3 Check Secrets
+List all secrets:
+```
+$ kubectl get secrets
+NAME                  TYPE                                  DATA      AGE
+default-token-m93m1   kubernetes.io/service-account-token   3         39d
+wlsecret              Opaque                                2         19m
+```
+
+### 5. Check Weblogic Server Status via Administrator Console
+The admin console URL is 'http://[hostIP]:30007/console'.
+
+### 6. Troubleshooting
+You can trace WebLogic server output and logs for troubleshooting.
+Trace WebLogic server output. Note you need to replace $serverPod with the actual pod name of a WebLogic server.
+```
+$ kubectl logs -f $serverPod
+```
+You can look at the WebLogic server logs by running:
+```
+$ kubectl exec managed-server-0 -- tail -f /u01/wlsdomain/servers/managed-server-0/logs/managed-server-0.log
+$ kubectl exec managed-server-0 -- tail -f /u01/wlsdomain/servers/managed-server-1/logs/managed-server-1.log
+$ kubectl exec managed-server-0 -- tail -f /u01/wlsdomain/servers/AdminServer/logs/AdminServer.log
+```
+
+### 7. Restart All Pods
+#### 7.1 Shutdown the Managed Servers' Pods Gracefully
+```
+$ kubectl exec -it managed-server-0 -- /u01/wlsdomain/bin/stopManagedWebLogic.sh managed-server-0 t3://admin-server:8001
+$ kubectl exec -it managed-server-1 -- /u01/wlsdomain/bin/stopManagedWebLogic.sh managed-server-1 t3://admin-server:8001
+```
+#### 7.2 Shutdown the Administrator Server Pod Gracefully
+First gracefully shutdown admin server process. Note that you need to replace $adminPod with the real admin server pod name.
+```
+$ kubectl exec -it $adminPod -- /u01/wlsdomain/bin/stopWebLogic.sh <username> <password> t3://localhost:8001
+```
+Next manually delete the admin pod.
+```
+$ kubectl delete pod/$adminPod
+```
+After the pods are stopped, each pod's corresponding controller is responsible for restarting the pods automatically.
+Wait until all pods are running and ready again. Monitor status of pods via `kubectl get pod`.
+
+### 8. Cleanup
+Run the script clean.sh to remove all resources from your Kubernetes cluster. You can also do the cleanup indivisually by running the following commands:
```

Comment:
Same thing here with `clean.sh`: it contains the exact same commands as below, so there is no value in having it. If you have a script, make it do validation and error checking as well.

Also, there is a spelling error: should be _individually_.